Resources/quirks.json

```diff
@@ -1,9 +1,34 @@
 {
-	"extended-torso-twist": {
+	"EXTENDED-TORSO-TWIST": {
 		"name": "Extended Torso Twist",
 		"description": "This BattleMech has a wide range of motion at the torso joint, allowing it a larger field of fire.",
 		"properties": {
 			"torsoTwistAngleMulti": 1.5
 		}
-	}
+	},
+	"UBM-REVENANT": {
+        "name": "Revenant",
+        "description": "The Revenant drone is a highly mobile 'Mech with improved heat management capabilities and sensors.",
+        "color": "(R=0.5,G=1.0,B=0.0,A=1)",
+        "properties": {
+            "jumpJetFuelBurnTimeMulti": 1.15,
+            "jumpJetInitialZVelocityMulti": 1.15,
+            "jumpJetHeatMulti": 0.8,
+			"topSpeedMulti": 1.2,

```

Comment:
Could you please put the speed bonus into its own quirk sth like:
```
"quadmech": {
  "name": "Quad 'Mech",
  .....,
  "properties": {
     "topSpeedMulti": 1.2,
     ...
   }
 }
 ```

Reply:
Sure. Please make my life easy and accept this PR and I'll deal with it in the next release.

Reply:
Actually, I'll just issue a new one. All good.